### PR TITLE
Add subscription link support to notification email verification template

### PR DESCRIPTION
**Pull request by paskal**
Address #560, the backend part.

When `NOTIFY_EMAIL_SUBSCRIBE_URL` environment variable is provided, instead of embedding token to email it will be included as a link.

With `SITE=http://127.0.0.1` and `NOTIFY_EMAIL_SUBSCRIBE_URL=/subscribe?token=`, link will look like `http://127.0.0.1/subscribe?token=<subscription_token>`. The token is not escaped and I'll escape it if it will be necessary when the frontend part will be written.

After the frontend counterpart will be created, `NOTIFY_EMAIL_SUBSCRIBE_URL` default should be set to newly created URL location.

That's how verification email looks like when subscribe_url is supplied:

image

### CODE REVIEW

**Comment by akellbl4** [comment]
I thought it well be smth like this
Screenshot 2020-01-20 at 16 12 49

Two methods at the same time.

**Comment by umputun** [comment]
I think we should keep both methods

**Comment by paskal** [comment]
Updated, please see new look below.

image

**Comment by umputun** [comment]
LGTM, but we can't merge it till UI side is ready

**Comment by umputun** [comment]
my bad, looks like `{{if .SubscribeURL}}` makes it safe to merge

File: README.md [hunk 1]
```diff
@@ -155,6 +155,7 @@ _this is the recommended way to run remark42_
 | notify.telegram.timeout | NOTIFY_TELEGRAM_TIMEOUT | `5s`                     | telegram timeout                                |
 | notify.email.fromAddress | NOTIFY_EMAIL_FROM      |                          | from email address                              |
 | notify.email.verification_subj | NOTIFY_EMAIL_VERIFICATION_SUBJ | `Email verification` | verification message subject          |
+| notify.email.subscribe-url | NOTIFY_EMAIL_SUBSCRIBE_URL | `` | URL parameters to add to SITE in order to include to subscription email instead of plain token |
```
Comment (umputun) [inline README.md]: I know what you meant, but the wording is confusing.

**Pull request merged**